.circleci/config.yml

```diff
@@ -271,7 +271,7 @@ jobs:
         command: echo 'all server tests passed!'
 
   # pytest the server with SQL Server ODBC Driver 17
-  test_server_mssql:
+  test_server_mssql_2019:
     docker:
     - image: *server_builder_image
     - image: sassela/mssql-2019-cu8-ubuntu-16.04:0.0.3
```

Comment:
Thanks so much for correcting this, I can see I forgot to make the corresponding code review changes in the OSS config file 😞 

There is a test failing on CI, which I believe is caused by a missing reference to `test_pg_env` here:

```diff
    - image: sassela/mssql-2019-cu8-ubuntu-16.04:0.0.3
    - image: hasura/postgres-13.0-alpine-postgis3:latest
+     <<: *test_pg_env
```

Let me know if you'd like me to commit this directly to the branch or leave it to you.